app/controllers/system/admin/get_help_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+class System::Admin::GetHelpController < System::Admin::Controller
+  def index
+    start_date = params[:start_date].presence
+    end_date = params[:end_date].presence
+    load_users(start_date, end_date)
+    @get_help_data = fetch_all_recent_live_feedbacks(start_date, end_date)
+    load_assessment_question_hash
+    load_course_user_hash
+  end
+
+  private
+
+  def load_users(start_date = nil, end_date = nil)
+    @users = if start_date.present? || end_date.present?
+               User.human_users.includes(:emails)
+             else
+               User.human_users.includes(:emails).active_in_past_7_days
+             end
+    @user_hash = @users.index_by(&:id)
+  end
```

Comment:
Is there really a need to grab all users like this? Shouldn't it only grab whichever users used Get Help in the date range?